X86/X86MachineInstructionRaiserUtils.cpp

```diff
@@ -542,7 +543,7 @@ X86MachineInstructionRaiser::find64BitSuperReg(unsigned int PhysReg) {
   }
 
   // Nothing to do if PhysReg is one of EFLAG bits, FPSW, FPCW
-  if (isEflagBit(PhysReg))
+  if (isEflagBit(PhysReg) || PhysReg == X86::EFLAGS)
```

Comment:
I'd expect that the client of this function is the right place to check validity of the PhysReg whose SuperReg manifestation is being asked for. See attached diff (based on this branch).
[116-suggested.txt](https://github.com/microsoft/llvm-mctoll/files/6978956/116-suggested.txt)

Reply:
Applied, thanks!